### PR TITLE
Adding VPN configuration details

### PR DESCRIPTION
**Pull request by sunasing**

### CODE REVIEW

**Comment by sunasing** [comment]
#sign-off

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 1]
```diff
@@ -62,9 +62,25 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
 > [!NOTE]
 > Defender for Endpoint for iOS supports creating custom indicators only for IP addresses and URLs/domains.
 
-## Web Protection
+## Web Protection and VPN
 
-By default, Defender for Endpoint for iOS includes and enables the web protection feature. [Web protection](web-protection-overview.md) helps to secure devices against web threats and protect users from phishing attacks.
+By default, Defender for Endpoint for iOS includes and enables the web protection feature. [Web protection](web-protection-overview.md) helps to secure devices against web threats and protect users from phishing attacks. Defender for Endpoint for iOS uses a local VPN in order to provide this protection.
+
+While enabled by default, there might be some cases that requires you to disable VPN. For example, you want to run some apps that do not work when a VPN is configured. In such cases, you can choose to disable VPN from the app on the device by following the steps below.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:69]: minor grammar detail
```suggestion
While enabled by default, there might be some cases that require you to disable VPN. For example, you want to run some apps that do not work when a VPN is configured. In such cases, you can choose to disable VPN from the app on the device by following the steps below:
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 2]
```diff
@@ -62,9 +62,25 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
...
+
+1. On your iOS device, open the **Settings** app and then click on **VPN**.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:71]: Suggest touch usage as well as mouse pointer:
```suggestion
1. On your iOS device, open the **Settings** app and then click or tap **VPN**.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 3]
```diff
@@ -62,9 +62,25 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
...
+1. Click on the "i" button for Microsoft Defender ATP.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:72]: (same as above)
```suggestion
1. Click or tap the "i" button for Microsoft Defender ATP.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 4]
```diff
@@ -62,9 +62,25 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
...
+1. Toggle-off **Connect On Demand** to disable VPN.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:73]: Suggestion for touch usage as well as mouse pointer usage:
```suggestion
1. On your iOS device, open the **Settings** app and click or tap  **VPN**.
1. Click or tap the "i" button for Microsoft Defender ATP.
1. Toggle off **Connect On Demand** to disable VPN.
```

File: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md [hunk 5]
```diff
@@ -62,9 +62,25 @@ Defender for Endpoint for iOS enables admins to configure custom indicators on i
...
+
+    > [!div class="mx-imgBorder"]
+    > ![VPN config connect on demand](images/ios-vpn-config.png)
+
+> [!NOTE]
+> Web Protection will not be available when VPN is disabled. To re-enable Web Protection, open Microsoft Defender for Endpoint app on the device and click on **Start VPN**.
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md:79]: (same as above)
```suggestion
> Web Protection will not be available when VPN is disabled. To re-enable Web Protection, open the Microsoft Defender for Endpoint app on the device and click or tap **Start VPN**.
```

**Comment by illfated** [comment]
Looks good to me otherwise.

**Pull request merged**